tools/moFF/moff.xml

```diff
@@ -160,7 +160,17 @@
                 --mbr $task.mbr
                 --xic_length $task.xic_length
                 --rt_peak_win $task.rt_peak_win
-                --loc_out ./out
+				--loc_out ./out
```

Comment:
@ieguinoa  I added the filtering also in apex cases. I still have to test it. I just copied what I wrote the 'all' case